lib/membrane_element_rtp_aac/depayloader.ex

```diff
@@ -16,6 +17,9 @@ defmodule Membrane.RTP.AAC.Depayloader do
               ],
               channels: [
                 default: 2
+              ],
+              mode: [
+                spec: :lbr | :hbr
```

Comment:
As discussed - I think we can stay with that option as under normal circumstances information about the mode is put in some side channel (like SDP).